cmd/jujud/agent/machine/manifolds_test.go

```diff
@@ -53,6 +54,9 @@ func (*ManifoldsSuite) TestManifoldNames(c *gc.C) {
 		"fan-configurer",
 		"global-clock-updater",
 		"host-key-reporter",
+		"is-controller-flag",
+		"is-responsible-controller-flag",
```

Comment:
'is-responsible-controller-flag' still feels really hard for me to get my head around. responsible for what?

is-primary-controller
is master controller
is singleton controller

I don't know that I have anything better.

Reply:
Yeah, I just copied from the model worker, figured it was better to be consistent. I like "is-primary-controller-flag", will go with that.